c7n/resources/cw.py

```diff
@@ -945,13 +945,15 @@ class SubscriptionFilter(BaseAction):
                 filter_pattern: ip
                 destination_arn: arn:aws:logs:us-east-1:1234567890:destination:lambda
                 distribution: Random
+                role_arn: "arn:aws:iam::123456789012:role/testCrossAccountRole"
```

Comment:
for the example let's do {account_id} for the account value so it works across accounts.